tee/src/cli.rs

```diff
@@ -18,4 +18,5 @@ pub(crate) fn create_app<'a, 'b>() -> App<'a, 'b> {
                 .short("a")
                 .long("append"),
         )
+        .arg(Arg::with_name("ignore").help("Ignore interrupt signals.").short("i").long("ignore"))
```

Comment:
I think the long name usually are called `ignore-interrupts`, but if you think that `ignore` is enough, I'm ok with that as long put `ignore-interrupts` as a visible alias

Reply:
I went ahead and went with `ignore-interrupts`.